src/render/draw_heatmap.js

```diff
@@ -29,14 +29,14 @@ function drawHeatmap(painter: Painter, sourceCache: SourceCache, layer: HeatmapS
 
         // Allow kernels to be drawn across boundaries, so that
         // large kernels are not clipped to tiles
-        context.setStencilMode(StencilMode.disabled());
+        context.setStencilMode(StencilMode.disabled);
 
         bindFramebuffer(context, painter, layer);
 
         context.clear({ color: Color.transparent });
 
         // Turn on additive blending for kernels, which is a key aspect of kernel density estimation formula
-        context.setColorMode(util.extend(painter.colorModeForRenderPass(), {
+        context.setColorMode(util.extend({}, painter.colorModeForRenderPass(), {
```

Comment:
This is better written as `new ColorMode([gl.ONE, gl.ONE], Color.transparent, [true, true, true, true])` I think.